wordpress-mobile/WordPress-iOS
# Migrate CommentServiceRemote classes & models to WordPressKit

Fixes #7308 and #7307 

This PR:
* Migrates the `RemoteComment` model to the WPKit framework
* Migrates `CommentServiceRemote.h` to the WPKit framework
* Migrates `CommentServiceRemoteREST` to the WPKit framework
* Migrates `CommentServiceRemoteXMLRPC` to the WPKit framework

Ref #7301

**To test:**
1. Clean DerivedData.
2. Update CocoaPods repo with pod repo update.
3. Install the pods with pod install.
4. Run _all_ unit tests and make sure they are green.
5. Build & run the app:
a. verify comment management is working as expected on .com
b. verify comment management is working are working as expected on .org

Needs review: @astralbodies
